library/public/interactor.h

```diff
@@ -234,24 +223,33 @@ class F3D_EXPORT interactor
   /**
    * Play a VTK interaction file.
    */
-  virtual bool playInteraction(const std::string& file) = 0;
+  virtual bool playInteraction(const std::string& file, double deltaTime = 1.0 / 30,
+    std::function<void()> userCallBack = nullptr) = 0;
 
   /**
    * Start interaction and record it all in a VTK interaction file.
    */
   virtual bool recordInteraction(const std::string& file) = 0;
 
   /**
-   * Start the interactor.
+   * Start the interactor event loop.
    * Make sure the window is initialized first, by calling `window.render()`
+   * The event loop will be triggered every deltaTime in seconds, and userCallBack will be called at
+   * the start of the event loop
    */
-  virtual void start() = 0;
+  virtual void start(double deltaTime = 1.0 / 30, std::function<void()> userCallBack = nullptr) = 0;
```

Comment:
it doesn't require Python binding changes?

Reply:
nope, the params are automatically wrapped